tsconfig.esm.json

```diff
@@ -0,0 +1,8 @@
+{
+  "extends": "./tsconfig.json",
+  "compilerOptions": {
+    "module": "ES6",
+    "declaration": false,
```

Comment:
Why false?

Reply:
Declarations are already generated from cjs build, setting it `true` would be unnecessary work.